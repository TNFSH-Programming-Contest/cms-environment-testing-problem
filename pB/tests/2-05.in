-522012888 -458498468
-827902558 -11335241
