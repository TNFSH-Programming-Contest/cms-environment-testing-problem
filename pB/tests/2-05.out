-1349915446 -469833709
